modules/diskq/diskq-global-metrics.h

```diff
@@ -27,5 +27,7 @@
 
 void diskq_global_metrics_init(void);
```

Comment:
I've checked and thanks to `disk_buffer_module_init()`, global metrics are not initialized when no disk-buffer is used in the configuration. 👍🏻